common-util/src/main/java/org/sunbird/common/models/util/JsonKey.java

```diff
@@ -646,6 +646,8 @@
   public static final String CASSANDRA_UPDATE_BATCH_SIZE = "cassandra_update_batch_size";
   public static final String ORG_EXTERNAL_ID = "orgExternalId";
   public static final String ORG_PROVIDER = "orgProvider";
+  public static final String EXTERNAL_IDS = "externalIds";
+  public static final String UPDATED_ON = "updatedOn";
```

Comment:
Why are we using the constant "updatedOn" for? If it is for audit purpose, we should use "lastUpdatedOn" consistently in all APIs.